<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimpleCalcView</class>
 <widget class="QWidget" name="SimpleCalcView">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>682</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>650</width>
    <height>682</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>650</width>
    <height>682</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Sans-serif</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Simple Calculator</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	background-color: #191A19;
	color: white;
	font-family: Sans-serif;
}

QLineEdit {
	border: 1px solid #606060;
	border-radius: 30px;
	padding: 20px;
}

QPushButton {
	background-color: #363333;
	width: 40px;
	height: 30px;
	border-radius: 15px;
}

QPushButton::hover {
	background-color: #444444;
}

QPushButton::pressed {
	background-color: #282828;
}

QDoubleSpinBox  {
	border: 1px solid #606060;
	border-radius: 30px;
	padding: 20px;
}</string>
  </property>
  <widget class="QPushButton" name="homeBtn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>640</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans-serif</family>
     <pointsize>13</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: #374045;
	border-radius: 15px;
}

QPushButton::hover {
	background-color: #3A4750;
}

QPushButton::pressed {
	background-color: #303841;
}</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
   <property name="iconSize">
    <size>
     <width>48</width>
     <height>48</height>
    </size>
   </property>
   <property name="autoRepeat">
    <bool>true</bool>
   </property>
   <property name="autoExclusive">
    <bool>true</bool>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>611</width>
     <height>401</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="btns_grid">
    <property name="leftMargin">
     <number>50</number>
    </property>
    <property name="rightMargin">
     <number>50</number>
    </property>
    <property name="horizontalSpacing">
     <number>6</number>
    </property>
    <property name="verticalSpacing">
     <number>2</number>
    </property>
    <item row="5" column="1">
     <widget class="QPushButton" name="x_btn">
      <property name="font">
       <font>
        <family>Sans-serif</family>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>x</string>
      </property>
      <property name="shortcut">
       <string>X</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="num_frou_btn">
      <property name="font">
       <font>
        <family>Sans-serif</family>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>4</string>
      </property>
      <property name="shortcut">
       <string>4</string>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QPushButton" name="num_three_btn">
      <property name="font">
       <font>
        <family>Sans-serif</family>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>3</string>
      </property>
      <property name="shortcut">
       <string>3</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QPushButton" name="num_six_btn">
      <property name="font">
       <font>
        <family>Sans-serif</family>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>6</string>
      </property>
      <property name="shortcut">
       <string>6</string>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QPushButton" name="mul_btn">
      <property name="font">
       <font>
        <family>Sans-serif</family>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>*</string>
      </property>
      <property name="shortcut">
       <string>*</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="clear_btn">
      <property name="font">
       <font>
        <family>Sans-serif</family>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="text">
       <string extracomment="Clear">C</string>
      </property>
      <property name="shortcut">
       <string>Esc</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="open_bracket_btn">
      <property name="font">
       <font>
        <family>Sans-serif</family>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>(</string>
      </property>
      <property name="shortcut">
       <string>(</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="num_one_btn">
      <property name="font">
       <font>
        <family>Sans-serif</family>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>1</string>
      </property>
      <property name="shortcut">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="num_eight_btn">
      <property name="font">
       <font>
        <family>Sans-serif</family>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>8</string>
      </property>
      <property name="shortcut">
       <string>8</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QPushButton" name="undo_btn">
      <property name="font">
       <font>
        <family>Sans-serif</family>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>CE</string>
      </property>
      <property name="iconSize">
       <size>
        <width>27</width>
        <height>27</height>
       </size>
      </property>
      <property name="shortcut">
       <string>Backspace</string>
      </property>
      <property name="autoRepeat">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="4">
     <widget class="QPushButton" name="minus_btn">
      <property name="font">
       <font>
        <family>Sans-serif</family>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>-</string>
      </property>
      <property name="shortcut">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QPushButton" name="num_nine_btn">
      <property name="font">
       <font>
        <family>Sans-serif</family>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>9</string>
      </property>
      <property name="shortcut">
       <string>9</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QPushButton" name="null_btn">
      <property name="font">
       <font>
        <family>Sans-serif</family>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="shortcut">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="num_five_btn">
      <property name="font">
       <font>
        <family>Sans-serif</family>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>5</string>
      </property>
      <property name="shortcut">
       <string>5</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="sin_btn">
      <property name="font">
       <font>
        <family>Sans-serif</family>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string notr="true">sin</string>
      </property>
      <property name="shortcut">
       <string>S</string>
      </property>
     </widget>
    </item>
    <item row="5" column="4">
     <widget class="QPushButton" name="equal_btn">
      <property name="font">
       <font>
        <family>Sans-serif</family>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #E16428;
}

QPushButton::hover {
	background-color: #ff8000;
}

QPushButton::pressed {
	background-color: #b35900;
}</string>
      </property>
      <property name="text">
       <string>=</string>
      </property>
      <property name="shortcut">
       <string>Return</string>
      </property>
      <property name="autoRepeat">
       <bool>true</bool>
      </property>
      <property name="autoExclusive">
       <bool>false</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QPushButton" name="div_btn">
      <property name="font">
       <font>
        <family>Sans-serif</family>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>/</string>
      </property>
      <property name="shortcut">
       <string>/</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QPushButton" name="ln_btn">
      <property name="font">
       <font>
        <family>Sans-serif</family>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string notr="true">ln</string>
      </property>
      <property name="shortcut">
       <string>T</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QPushButton" name="mod_btn">
      <property name="font">
       <font>
        <family>Sans-serif</family>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string notr="true">mod</string>
      </property>
      <property name="shortcut">
       <string>T</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="cos_btn">
      <property name="font">
       <font>
        <family>Sans-serif</family>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string notr="true">cos</string>
      </property>
      <property name="shortcut">
       <string>C</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="reverse_func_btn">
      <property name="font">
       <font>
        <family>Sans-serif</family>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string notr="true">arc</string>
      </property>
      <property name="shortcut">
       <string>T</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="log_btn">
      <property name="font">
       <font>
        <family>Sans-serif</family>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string notr="true">log</string>
      </property>
      <property name="shortcut">
       <string>T</string>
      </property>
     </widget>
    </item>
    <item row="5" column="3">
     <widget class="QPushButton" name="dot_btn">
      <property name="font">
       <font>
        <family>Sans-serif</family>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>.</string>
      </property>
      <property name="shortcut">
       <string>.</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="close_bracket_btn">
      <property name="font">
       <font>
        <family>Sans-serif</family>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>)</string>
      </property>
      <property name="shortcut">
       <string>)</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="sqrt_btn">
      <property name="font">
       <font>
        <family>Sans-serif</family>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string notr="true">sqrt</string>
      </property>
      <property name="shortcut">
       <string>T</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QPushButton" name="pow_btn">
      <property name="font">
       <font>
        <family>Sans-serif</family>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string notr="true">^</string>
      </property>
      <property name="shortcut">
       <string>^</string>
      </property>
     </widget>
    </item>
    <item row="4" column="4">
     <widget class="QPushButton" name="plus_btn">
      <property name="font">
       <font>
        <family>Sans-serif</family>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="whatsThis">
       <string/>
      </property>
      <property name="accessibleDescription">
       <string>Plus</string>
      </property>
      <property name="text">
       <string extracomment="Plus">+</string>
      </property>
      <property name="shortcut">
       <string>+</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="num_seven_btn">
      <property name="font">
       <font>
        <family>Sans-serif</family>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>7</string>
      </property>
      <property name="shortcut">
       <string>7</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="tan_btn">
      <property name="font">
       <font>
        <family>Sans-serif</family>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string notr="true">tan</string>
      </property>
      <property name="shortcut">
       <string>T</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QPushButton" name="num_two_btn">
      <property name="font">
       <font>
        <family>Sans-serif</family>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>2</string>
      </property>
      <property name="shortcut">
       <string>2</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="replaceLabel">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>140</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans-serif</family>
     <pointsize>15</pointsize>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>x value</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_edit">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>541</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans-serif</family>
     <pointsize>16</pointsize>
     <italic>false</italic>
     <bold>true</bold>
     <stylestrategy>PreferAntialias</stylestrategy>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>255</number>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string>0</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="replaceSpin">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>130</y>
     <width>191</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans-serif</family>
     <pointsize>14</pointsize>
     <stylestrategy>PreferAntialias</stylestrategy>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhFormattedNumbersOnly</set>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="minimum">
    <double>-1000000000.000000000000000</double>
   </property>
   <property name="maximum">
    <double>1000000000.000000000000000</double>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
